Makefile

```diff
@@ -0,0 +1,50 @@
+.DEFAULT_GOAL := all
+isort = isort -rc -l 120 qcelemental
+black = black -S -l 120 --target-version py37 qcelemental --exclude "checkup_data|data"
```

Comment:
kill Makefile before merge

Reply:
My plan was to actually consider carrying around a Makefile for this kind of `make format` kind of operations. I have seen it in other projects and like it, but can drop it.

Reply:
ah, ok. I supposed the Makefile to have been experimental and added by mistake.